gitlab-cpu-pipeline.yml

```diff
@@ -42,13 +42,15 @@ ci_builder_pre:
         - pip3 install setuptools
         - pip3 install sphinx
         - pip3 install sphinx_rtd_theme
+        - pip3 install 'seissolxdmf>=0.1.0'
+        - pip3 install h5py
         - tar -xzf netcdf-4.6.1.tar.gz
         - cd netcdf-4.6.1 && CC=h5pcc ./configure --prefix=/usr --enable-shared=no --disable-dap && make && sudo make install && cd ..
         - unzip libxsmm-1.17.zip
         - cd libxsmm-1.17 && make generator && sudo cp bin/libxsmm_gemm_generator /usr/bin && cd ..
         - git clone https://github.com/uphoffc/ImpalaJIT.git &&
           mkdir impala_build && cd impala_build && cmake ../ImpalaJIT && sudo make -j $(nproc) install && cd ..
-        - git clone --depth 1 --branch v1.0.0 https://github.com/SeisSol/easi.git &&
```

Comment:
Maybe it is better to specify a specific version? What do you think or there is a reason to not to do it?

Reply:
I just wanted to use the latest version of easi. We can specify a specific version here, but then we might need to adapt the pipeline script, when there is an easi update.